src/renderer/components/about/index.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="about-dialog" :class="theme">
+    <el-dialog
+      :visible.sync="showAboutDialog"
+      :show-close="false"
+      :modal="true"
+      custom-class="ag-dialog-table"
+      width="400px"
+    >
+      <img class="logo" :src="markTextLogo" height="128" width="128" />
+      <el-row>
+        <el-col :span="24">
+          <h3 class="text fg-color-dark">{{ name }}</h3>
+        </el-col>
+        <el-col :span="24">
+          <div class="text">{{ versionPrefix }}{{ version }}</div>
+        </el-col>
+        <el-col :span="24">
+          <div class="text">Copyright © 2018 Jocs</div>
+        </el-col>
+      </el-row>
+  </div>
+</template>
+
+<script>
+  import { remote } from 'electron'
+  import { mapState } from 'vuex'
+  import bus from '../../bus'
+
+  export default {
+    data () {
+      return {
+        showAboutDialog: false,
+        name: 'Mark Text',
+        version: remote.app.getVersion(),
+        versionPrefix: 'v',
+        markTextLogo: require('../../../../static/logo-small.png')
```

Comment:
put `logo-small.png` in the path `src/render/assets/images/logo-small.png`, and don't need to `require` or `import`, just write it to the `src` of `img` tag.

BTY, the size of `logo-small.png` is too big, I think 150 * 150 px is enough, can you resize it, and rename it to `logo.png`

Reply:
The current size is 128x128x, should it be changed anyway?

Reply:
I mean the nature size of the image, not the width and height which we set to the style.